### PR TITLE
Update season preview

### LINKED ISSUE #117: Evaluate amount of cards in a row (1920x1080 res)
**Question**
On a full HD display, the current amount of cards in a row is 4.
When on something bigger than Full HD (like WQHD or 4K), the amount is 6.

For some people, 4 cards might lead the sizes of the cards to increase too much.

**Task**
Find out, if 6 cards for Full HD displays is feasible and is not proving disadvantageous.

**Your opinion**
Do you think 4 cards is not enough? Do you think 6 could fit?

Please leave your comments here or in Discord in the #feedback channel.

### PR DESCRIPTION
**Pull request by NicoAiko**
Season Preview now adds the var changes to URL query.
Season Preview now supports 5 cards in a row.
Add check mark for already added media and remove
disabled property.
Show card actions only when authenticated.
Show studios in name.

Language translations necessary.

Fixes #117 #112

### CODE REVIEW

File: src/assets/locales/zh_CN.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "englishReading": "Simplified Chinese",
   "actions": {
     "add": "添加",
+    "added": "Added to the list",
```
Comment (Mornwind) [inline src/assets/locales/zh_CN.json]:
```suggestion
    "added": "已添加至列表",
```

Comment (Mornwind) [inline src/assets/locales/zh_CN.json]:
```suggestion
    "added": "已添加至列表",
```

File: src/assets/locales/fr.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
   "englishReading": "French",
   "actions": {
     "add": "Ajouter",
+    "added": "Added to the list",
```
Comment (TheRaphael0000) [inline src/assets/locales/fr.json]: Ajouté à la liste

Comment (TheRaphael0000) [inline src/assets/locales/fr.json]: Ajouté à la liste

File: src/assets/locales/ar.json [hunk 3]
```diff
@@ -3,6 +3,7 @@
   "englishReading": "Arabic",
   "actions": {
     "add": "اضف",
+    "added": "Added to the list",
```
Comment (nattadasu) [inline src/assets/locales/ar.json]:
```suggestion
    "added": "يضاف إلى القائمة",
```

File: src/assets/locales/id.json [hunk 4]
```diff
@@ -3,6 +3,7 @@
   "englishReading": "Indonesian",
   "actions": {
     "add": "Tambah",
+    "added": "Added to the list",
```
Comment (nattadasu) [inline src/assets/locales/id.json]:
```suggestion
    "added": "Ditambahkan ke dalam daftar",
```

File: src/assets/locales/ms.json [hunk 5]
```diff
@@ -3,6 +3,7 @@
   "englishReading": "Malay",
   "actions": {
     "add": "Tambah",
+    "added": "Added to the list",
```
Comment (nattadasu) [inline src/assets/locales/ms.json]:
```suggestion
    "added": "Ditambah ke senarai",
```

File: src/assets/locales/pt_BR.json [hunk 6]
```diff
@@ -3,6 +3,7 @@
   "englishReading": "Portuguese (Brazil)",
   "actions": {
     "add": "Adicionar",
+    "added": "Added to the list",
```
Comment (hlorenzi) [inline src/assets/locales/pt_BR.json]:
```suggestion
    "added": "Adicionado à lista",
```

**Pull request merged**